op-node/rollup/sequencing/origin_selector.go

```diff
@@ -114,6 +117,22 @@ func (los *L1OriginSelector) CurrentAndNextOrigin(ctx context.Context, l2Head et
 	los.mu.Lock()
 	defer los.mu.Unlock()
 
+	if los.recoverMode {
+		currentOrigin, err := los.l1.L1BlockRefByHash(ctx, l2Head.L1Origin.Hash)
+		if err != nil {
+			return eth.L1BlockRef{}, eth.L1BlockRef{},
+				derive.NewTemporaryError(fmt.Errorf("failed to fetch current L1 origin: %w", err))
+		}
+		los.currentOrigin = currentOrigin
+		nextOrigin, err := los.l1.L1BlockRefByNumber(ctx, currentOrigin.Number+1)
+		if err != nil {
+			return eth.L1BlockRef{}, eth.L1BlockRef{},
+				derive.NewTemporaryError(fmt.Errorf("failed to fetch next L1 origin: %w", err))
+		}
+		los.nextOrigin = nextOrigin
```

Comment:
would it be redundant here to info-log that the l1 origin selector is in recovery mode and attach the current and next origin, and l2head?